README.md

```diff
@@ -40,31 +40,42 @@ east student silly already breeze enact seat trade few way online skin grass hum
   <summary>How to generate a root private key</summary>
 
 ```
-$ cardano-address recovery-phrase generate --size 15 > recovery-phrase.prv
-$ cat recovery-phrase.prv | cardano-address key from-recovery-phrase Shelley
-xprv1fzu4e8cecxshgzzxzh7557sd8tffqreeq2je7fgsm7f02mq849vdupw7qwgxc3qawyqev0l8ew0f4fkp8hvr8mskz4hz6e6ejzjlevcskcl6lqpr07u7552fsfgteztuclse7luh4cp493zdhkrjdss0250cdw8n
+$ cat recovery-phrase.txt | cardano-address key from-recovery-phrase Shelley
```

Comment:
Why this change ? I think it was clearer before with both instructions visible in the same excerpt.

Reply:
@KtorZ  fair point. I was thinking here in the following way. We use `prv`, `pub`, `xpub`, `pub` for keys...and for me mnemonic sequence is just text.... If that does not persuade you I will revert this in coming commit. @rvl do you have an opinion on that?

Reply:
Yes and yes
1. It would be better if the `recovery-phrase generate` command were added back to provide context.
2. `.txt` is a good extension to describe the content of the file.